### DISCUSSION TITLE
New Blogpost - WebRTC Android SDK: Build Your App in 4 Simple Steps!

### DISCUSSION THREAD

**Discussion by burak-58** [ant-media/Ant-Media-Server#6005] [Announcements]
Hi my friends,
We have published the blogpost that tells how you can create an Android WebRTC streaming application easily. You can reach it [here](https://antmedia.io/building-android-app-using-webrtc-android-sdk/).

Our main goal while writing these posts is making life easy fpr streaming project creators. So they should be useful, easy to understand and follow. 

I am open to the feedback and/or feedforward for this and possible new posts.

**Comment by jmax07** [comment]
It would be great to have the option of being able to do pre-recorded lives

**Reply by yashtandon113** [reply]
@jmax07 
If you want to stream the pre-recorded content, then you can do it with the help of Ant Media Server's playlist feature. Please check out the following document: https://antmedia.io/docs/guides/publish-live-stream/playlist/

Please let me know if I misunderstand your question.

**Comment by nocpbxchap** [comment]
Is not working , check your sample code. I build apk start running but nothing happen.

**Reply by burak-58** [reply]
Thank you @nocpbxchap,
Do you have any logs to share with me?
